app/api.py

```diff
@@ -509,7 +509,7 @@ def write_status(self, type, written, skipped, total, id_field):
 
 
 class Search(BaseApiHandler):
-    https_required = False
+    https_required = True
```

Comment:
I wonder if this can break existing clients, but it would probably be safe because our document points to HTTPS URLs?:
https://github.com/google/personfinder/wiki/DataAPI

We can probably just launch it to see if anyone complains.

Reply:
Yeah, and I'd be kinda surprised if someone couldn't handle making HTTPS requests these days. If someone does break though, this should be an easy PR to roll back.

Reply:
I believe it is still common that it requires an additional step to support HTTPS in HTTP client libraries, like pointing to the path to SSL certificates in the system. It is also possible that the request is not configured to handle HTTP redirects. Anyway hopefully the clients have specified HTTPS URLs from the beginning.